### PR TITLE
[doc] Update Contributing section

### PR DESCRIPTION
**Pull request by amoutenet**
Write "How to make a pull request section" + minor updates in "How to contribute to the documentation"

### CODE REVIEW

File: doc/contributing/documentation.rst [hunk 1]
```diff
@@ -16,17 +16,17 @@ If you are willing to contribute to this project, we however ask you to take the
 - Check our `Tutorials on github <https://github.com/TRIQS/tutorials>`_
 - Respect our :ref:`documentation conventions <conventions>`
 
-If you have any question, feel free to use our `Google group <https://triqs.github.io/announcements>`_.
+As for code contribution, you are going to submit your changes as a pull request from the unstable branch
```
Comment (Wentzell) [inline doc/contributing/documentation.rst]:
```suggestion
As for code contribution, you are going to submit your changes as a pull request to the unstable branch 
```

File: doc/contributing/pull_request.rst [hunk 2]
```diff
@@ -1,8 +1,79 @@
 .. _pull_request:
 
-.. warning::
-     Careful: This part of the documentation is currently being written
-
 How to make a pull request?
 ===========================
 
+You will need to submit your contributions in the form of pull requests to the unstable branch of the TRIQS repository,
+and this page is here to guide you through this process
```
Comment (Wentzell) [inline doc/contributing/pull_request.rst]:
```suggestion
and this page is here to guide you through this process.
```

File: doc/contributing/pull_request.rst [hunk 3]
```diff
@@ -1,8 +1,79 @@
...
+
+     git push my_triqs my_first_PR
+     
+Please keep in your commit messages clear and
+concise, using brackets to specify what you are modifying (for instance [doc] if you contribute to the documentation, [det_manip] if you fix a bug
+in the det_manip class, etc.). Do not multiply your commits, or squash them before doing your pull request
+(see for example `this page <https://davidwalsh.name/squash-commits-git>`_).
+
+When you consider your pull request ready to be reviewed by TRIQS developers, go to the `triqs github <https://github.com/TRIQS/triqs>`_ and
+go to `Pull requests`, and `New pull request`. Be sure to select unstable as the base branch.
```
Comment (Wentzell) [inline doc/contributing/pull_request.rst]:
```suggestion
and click `New pull request`. Be sure to select unstable as the target branch and my_first_PR as the source branch.
```

File: doc/contributing/pull_request.rst [hunk 4]
```diff
@@ -1,8 +1,79 @@
...
+You can now work on this branch as usual, modifying files and committing your changes. Keep in mind to push your branch to your fork::
+
+     git push my_triqs my_first_PR
+     
+Please keep in your commit messages clear and
+concise, using brackets to specify what you are modifying (for instance [doc] if you contribute to the documentation, [det_manip] if you fix a bug
+in the det_manip class, etc.). Do not multiply your commits, or squash them before doing your pull request
+(see for example `this page <https://davidwalsh.name/squash-commits-git>`_).
+
+When you consider your pull request ready to be reviewed by TRIQS developers, go to the `triqs github <https://github.com/TRIQS/triqs>`_ and
```
Comment (Wentzell) [inline doc/contributing/pull_request.rst]:
```suggestion
When you consider your pull request ready to be reviewed by TRIQS developers, go to the github page of your fork
```

**Review by Wentzell — CHANGES_REQUESTED** [review]
Thank you Alice! This looks great!
I just had a few minor remarks.

**Comment by amoutenet** [comment]
@Wentzell thank you for the review, I applied the changes!

**Pull request merged**